Python 3.7.2 (bundled)
>>> l1 = []
>>> l1
[]
>>> type(l1)
<class 'list'>
>>> dir(l1)
['__add__', '__class__', '__contains__', '__delattr__', '__delitem__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__gt__', '__hash__', '__iadd__', '__imul__', '__init__', '__init_subclass__', '__iter__', '__le__', '__len__', '__lt__', '__mul__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__reversed__', '__rmul__', '__setattr__', '__setitem__', '__sizeof__', '__str__', '__subclasshook__', 'append', 'clear', 'copy', 'count', 'extend', 'index', 'insert', 'pop', 'remove', 'reverse', 'sort']
>>> len(l1)
0
>>> l1 = [4, 'hello wolrd', 0.37438, 'a', [1,3,5] ]
>>> l1
[4, 'hello wolrd', 0.37438, 'a', [1, 3, 5]]
>>> l1[1]
'hello wolrd'
>>> l1[4]
[1, 3, 5]
>>> l1[4][2]
5
>>> def tambah(a,b):
    return a+b

>>> type(tambah)
<class 'function'>
>>> l1.insert(tambah)
Traceback (most recent call last):
  File "<pyshell>", line 1, in <module>
TypeError: insert() takes exactly 2 arguments (1 given)
>>> l1
[4, 'hello wolrd', 0.37438, 'a', [1, 3, 5]]
>>> l1.append(tambah)
>>> l1
[4, 'hello wolrd', 0.37438, 'a', [1, 3, 5], <function tambah at 0x02FE8930>]
>>> l1[5]
<function tambah at 0x02FE8930>
>>> l1[5](34,5)
39
>>> l1[5](34,5)+l1[0]*l2[2]
Traceback (most recent call last):
  File "<pyshell>", line 1, in <module>
NameError: name 'l2' is not defined
>>> l1[5](34,5)+l1[0]*l1[2]
40.49752
>>> len(l1)
6
>>> l1[3] = "Hiiiii...."
>>> l1
[4, 'hello wolrd', 0.37438, 'Hiiiii....', [1, 3, 5], <function tambah at 0x02FE8930>]
>>> l1.reverse()
>>> l1
[<function tambah at 0x02FE8930>, [1, 3, 5], 'Hiiiii....', 0.37438, 'hello wolrd', 4]
>>> l1.sort()
Traceback (most recent call last):
  File "<pyshell>", line 1, in <module>
TypeError: '<' not supported between instances of 'list' and 'function'
>>> l2 = range(2,10)
>>> l2
range(2, 10)
>>> l2 = [2,3,4,5,6,7,8,9,10]
>>> l2
[2, 3, 4, 5, 6, 7, 8, 9, 10]
>>> l2.sort()
>>> l2
[2, 3, 4, 5, 6, 7, 8, 9, 10]
>>> l2.sort(reverse=True)
>>> l2
[10, 9, 8, 7, 6, 5, 4, 3, 2]
>>> l2.reverse()
>>> l2
[2, 3, 4, 5, 6, 7, 8, 9, 10]
>>> l2.reverse()
>>> l2
[10, 9, 8, 7, 6, 5, 4, 3, 2]
>>> l2
[10, 9, 8, 7, 6, 5, 4, 3, 2]
>>> l2[0]=5
>>> l2
[5, 9, 8, 7, 6, 5, 4, 3, 2]
>>> l2.remove(5)
>>> l2
[9, 8, 7, 6, 5, 4, 3, 2]
>>> l2.remove(5)
>>> l2
[9, 8, 7, 6, 4, 3, 2]
>>> l2.index(4)
4
>>> l2.index(1)
Traceback (most recent call last):
  File "<pyshell>", line 1, in <module>
ValueError: 1 is not in list
>>> 1 in l2
False
>>> 7 in l2
True
>>> 1 in l2 and l2.index(1) or -1
-1
>>> 7 in l2 and l2.index(7) or -1
2
>>> l2
[9, 8, 7, 6, 4, 3, 2]
>>> l2[9]
Traceback (most recent call last):
  File "<pyshell>", line 1, in <module>
IndexError: list index out of range
>>> l2
[9, 8, 7, 6, 4, 3, 2]
>>> l2[1]
8
>>> l2[1:3]
[8, 7]
>>> l2[1:4]
[8, 7, 6]
>>> l2[1:4]
[8, 7, 6]
>>> l2[1:4:-1]
[]
>>> l2[1:4]
[8, 7, 6]
>>> l2[-1]
2
>>> l2[6]
2
>>> l2[6] == l2[-1]
True
>>> l2[::-1]
[2, 3, 4, 6, 7, 8, 9]
>>> l2[:3]
[9, 8, 7]
>>> l2[:3:-1]
[2, 3, 4]
>>> l2[:3:-2]
[2, 4]
>>> l2[::-2]
[2, 4, 7, 9]
>>> 
